functions/__bass.py

```diff
@@ -23,7 +23,7 @@ def gen_script():
 
     # Use the following instead of /usr/bin/env to read environment so we can
     # deal with multi-line environment variables (and other odd cases).
-    env_reader = "python -c 'import os,json; print json.dumps({k:v for k,v in os.environ.iteritems()})'"
+    env_reader = "python -c 'import os,json; print(json.dumps({k:v for k,v in os.environ.items()}))'"
```

Comment:
`iteritems()` and `items()` are not quite equivalent, as in the latter (in Python 2 AFAIK) everything
goes in memory, but it should not be an issue here unless your env has like 10k entries.